.github/workflows/monitor-stable-releases.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Monitor-stable-releases
+
+#on:
+#  release:
+#    types: [released]
+
+
+concurrency: # This keeps multiple instances of the job from running concurrently for the same ref and event type.
+  group: release-${{ github.ref }}-${{ github.event_name }}
```

Comment:
This needs a different concurrency group name, the current one matches the release workflow, which is not something we want.

Reply:
Addresed 
 with: 
 https://github.com/netdata/netdata/pull/16366/commits/00651c37dc52f3d352cac9399f30a0b86eaab52b
 https://github.com/netdata/netdata/pull/16366/commits/f3cbc023843df84485d1b66aaa9f8b2998734c27